app/uk/gov/hmrc/selfservicetimetopay/controllers/SelfServiceTimeToPayController.scala

```diff
@@ -34,6 +39,25 @@ class SelfServiceTimeToPayController extends TimeToPayController {
   def submit: Action[AnyContent] = Action { implicit request =>
     Redirect(routes.EligibilityController.start())
   }
+  def getSignInQuestion: Action[AnyContent] = Action.async { implicit request =>
+    sessionCache.get.map {
+      case Some(ttpData@TTPSubmission(_, _, _, _, typeOfTax@Some(_), _, _, _, _)) =>
+        Ok(sign_in_question(EligibilityForm.typeOfTaxForm.fill(typeOfTax.get), ttpData.taxpayer.isDefined))
```

Comment:
Why is whether taxpayer is defined being passed into the sign in page? This probably exists from when the form was copied over and may need to go